packages/helpers/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "@actions/helpers",
+  "version": "1.0.0",
+  "description": "Helpers for creating actions",
+  "main": "lib/helpers.js",
+  "types": "lib/helpers.d.ts",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib",
+    "!.DS_Store"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "audit-moderate": "npm install && npm audit --json --audit-level=moderate > audit.json",
+    "test": "echo \"Error: run tests from root\" && exit 1",
+    "tsc": "tsc -p tsconfig.json"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/actions/toolkit.git",
+    "directory": "packages/helpers"
+  },
+  "keywords": [
+    "github",
+    "actions",
+    "helpers"
+  ],
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/actions/toolkit/issues"
+  },
+  "homepage": "https://github.com/actions/toolkit#readme",
```

Just for consistency with other toolkit packages 
```suggestion
  "homepage": "https://github.com/actions/toolkit/tree/main/packages/helpers",
```